### ISSUE TITLE
Resetting account breaks journal

### ISSUE THREAD

**Issue by aarondoet** [monicahq/monica#6729]
**Describe the bug**
I just set up Monica, wrote two journal entries and imported contacts. I messed something up while importing contacts so I thought I'd just reset my account in settings, no big deal. When I then wrote those journal entries again they did not show up anymore. Connecting to the database and looking through the existing tables and their contents I found out, that in `journal_entries` there were six rows (I made few more entries for testing) with id 1 to 6 and in `entries` there were only four rows with id 3 to 6. I was unsure whether that is the problem but after checking the logs of the monica docker container there were error messages looking like it is trying to find non-existant journal entries. So I just deleted the `journal_entries` rows with id 1 and 2 and indeed, it was working again.

TL;DR: Apparently when resetting an account, the journal entry contents get deleted from the `entries` table, but not the journal entries themselves in the `journal_entries` table.

**Which version are you using:**
 latest self hosted version using docker compose and the `monica:apache` image
